source/fe/fe_face.cc

```diff
@@ -956,7 +956,7 @@ FE_FaceP<dim, spacedim>::get_subface_interpolation_matrix(
           for (unsigned int k = 0; k < face_quadrature.size(); ++k)
             {
               const Point<dim - 1> p =
-                numbers::invalid_unsigned_int != 0u ?
+                subface != numbers::invalid_unsigned_int ?
```

Comment:
How the heck did we not get a tautological comparison warning here?

Reply:
Also - I think this should be `==` here, like on line 938.

Reply:
So much for it being obvious :-( I fixed it, thanks for noticing!